apps/dashboard/app/lib/account_cache.rb

```diff
@@ -127,13 +130,7 @@ def unique_qos_names
 
   # do you have _any_ account that can submit to this queue?
   def blocked_queue?(queue)
-    allow_accounts = queue.allow_accounts
-
-    if allow_accounts.nil?
-      false
-    else
-      allow_accounts.intersection(account_names).empty?
-    end
+    !(accounts.select {|account| account_allowed?(queue, account)}).any?
```

Comment:
I think instead of `!any?` you can use `none?`. Typically ruby has inverses like that (select vs. reject and so on)